init/lmod_bash_completions

```diff
@@ -168,7 +177,16 @@ _ml() {
 		COMPREPLY=( $(IFS=: compgen -W "${MODULEPATH}" -- "${cur}") )
 		;;
 	use|*-a*)
-		_filedir -d || return 0
+		_filedir -d 2>/dev/null || {
+			COMPREPLY=( $(compgen -d -- "${cur}") )
+			if [[ ${#COMPREPLY[@]} -eq 1 ]]; then
+				local dir=${COMPREPLY[0]}
+				if [[ ${dir} != "*/" ]]; then
+					COMPREPLY[0]="${dir}/"
+					compopt -o nospace
+				fi
+			fi
```

Comment:
@pneerincx wouldn't it be cleaner to define a dedicated function for this, rather than copy-pasting it?